site/tests/integration/block-publish-npm.test.ts

```diff
@@ -1,118 +1,121 @@
-import { Page } from "playwright";
-import { expect, test } from "playwright-test-coverage";
+import { test } from "playwright-test-coverage";
 
-import { publishBlock } from "../shared/blocks";
-import { resetDb } from "../shared/fixtures";
-import { login } from "../shared/nav";
+test("TODO: uncomment this file when NPM_PUBLISHING feature is released", () => {});
```

Comment:
Couldn't we set the environment variable in the test to make sure we don't break the feature?

Reply:
Do you mean conditionally skipping every test depending on the env variable, instead of commenting out tests? @CiaranMn

Reply:
I meant just [set the environment variable](https://nextjs.org/docs/basic-features/environment-variables#test-environment-variables) so that the feature appears when running tests so that we keep testing as if the feature was released – that way we don't make any changes that break it without realising.